src/version.ts

```diff
@@ -0,0 +1,32 @@
+import { existsSync, readFileSync } from 'node:fs'
+import { join } from 'node:path'
+
+export type BuildInfo = {
+  name: string
+  version: string
+  gitSha: string
+  buildTime: string
+}
+
+function readJsonIfExists<T = unknown>(path: string): T | null {
+    if (!existsSync(path)) {
+        return null
+    }
+
+    try {
+        return JSON.parse(readFileSync(path, 'utf8')) as T
+    } catch {
+        return null
+    }
+}
+
+const root = join(__dirname, '..')
+const build = readJsonIfExists<Partial<BuildInfo>>(join(root, 'build-info.json'))
+const pkg = readJsonIfExists<any>(join(root, 'package.json'))
```

Comment:
Here as well, we could just use `import * as pkg from '../package.json'`

Reply:
Thanks! I've [updated](https://github.com/fingerprintjs/dx-team-mock-for-proxy-integrations-e2e-tests/pull/47/commits/f7f464ff83f10cec76aad0d60ac851b6ebdaa9f8) the file.